topics/transcriptomics/tutorials/mirna-target-finder/tutorial.md

```diff
@@ -0,0 +1,726 @@
+> ![Experimental design](../../images/salmon_quasialignment.png "Quasi mapping")
+> The search procedure then advances to position i+ℓ−k in the read, and again begins hashing the k-mers it encounters. This process of determining the MMP and NIP of each processed k-mer and advancing to the NIP in the read continues until the NIP exceeds position l<sub>r</sub>−k where l<sub>r</sub> is the length of the read r. The result of applying this procedure to a read is a set S of query positions, MMP orientations and SA intervals, with one such triplet corresponding to each MMP.  ({% cite Srivastava2016 %}).
+{: .comment}
+
+
+> ### {% icon hands_on %} Hands-on: Quantify gene expression with Salmon
+>
+> 1. {% tool [Salmon quant](toolshed.g2.bx.psu.edu/repos/bgruening/salmon/salmon/0.14.1.2+galaxy1) %} with the following parameters:
+>    - *"Select salmon quantification mode:"*: `Reads`
+>        - *"Select a reference transcriptome from your history or use a built-in index?"*: `Use one from the history`
+>            - In *"Salmon index"*:
+>                - {% icon param-file %} *"Transcripts fasta file"*: `transcriptome_AtRTD2.fasta`
+>        - In *"Data input"*:
+>            - *"Is this library mate-paired?"*: `Single-end`
+>                - {% icon param-file %} *"FASTQ/FASTA file"*: `Control mRNA`
+>        - *"Validate mappings"*: `Yes`
+>    - {% icon param-file %} *"File containing a mapping of transcripts to genes"*: `annotation_AtRTD2.gtf`
+>
+> 3. Rename the outputs as `Salmon control mRNA (Quantification)` and `Salmon control mRNA (Gene Quantification)`
+> 2. Repeat the previous procedure by using the `BR treated mRNA` dataset
+>
+>   > ### {% icon comment %} Quasi-mapping sequence requeriments
+>   > Trimming the reads is not required  when using this method, since if there are k-mers in the reads that are not in the hash table, they are not counted. Quantification of the reads is only as good as the quality of the reference transcriptome.
+>   {: .comment}
+>
+{: .hands_on}
+
+Salmon generates two output from each read dataset: 
+
+- Quantification: summarizes the quantifications by transcript
+- Gene quantification: summarizes the quantification by gene
+
+Each output consist in a tabular dataset with five columns:
+
+- Name: the name of the target transcript provided in the input transcriptome
+- Length: the length of the target transcript
+- EffectiveLength: the computed effective length
```

Comment:
```suggestion
- Effective Length: the computed effective length
```

Reply:
In that case, both words appear joined in the Salmon output file.